### PR TITLE
authentication tests

### PR DESCRIPTION
**Pull request by jabajke**

### CODE REVIEW

File: tests/test_authentication.py [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+    assert data['refresh']
+
+
+@pytest.mark.django_db
+def test_user_login_fail(wrong_user_fixture, payload_fixture):
+    response = client.post(reverse('token_obtain_pair'), dict(
+        email=wrong_user_fixture.email,
+        password=payload_fixture['password']
+    ))
+    assert response.status_code == 401
```
Comment (shprexen) [inline tests/test_authentication.py:45]: use drf status code objects to make your code more obvious and readable.
`from rest_framework import status`

File: tests/test_authentication.py [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+    response = client.post(reverse('signup'), payload_fixture)
+    data = response.data
+
+    assert data['first_name'] == payload_fixture['first_name']
+    assert data['last_name'] == payload_fixture['last_name']
+    assert data['email'] == payload_fixture['email']
+    assert payload_fixture['password'] not in data.values()
+    assert payload_fixture['password2'] not in data.values()
+
+    del payload_fixture['password2']
```
Comment (shprexen) [inline tests/test_authentication.py:24]: why do we need that line?

**Review by shprexen — CHANGES_REQUESTED** [review]

File: tests/test_authentication.py (hunk 1 above)
Comment (jabajke) [reply tests/test_authentication.py]: done

File: tests/test_authentication.py (hunk 2 above)
Comment (jabajke) [reply tests/test_authentication.py:25]: signup endpoint has two password field, password and password2, just for confirmation, that user wrote correct password
so i have to add this value to dictionary to succeed, so after that i have to remove this value

**Review by shprexen — APPROVED** [review]

**Pull request merged**